src/components/ProfileUI/ProfileUI.jsx

```diff
@@ -1,16 +1,21 @@
 import React from "react";
 
-const ProfileUI = ({ name, surname }) => (
-  <div>
-    name: {name}
-    <br />
-    surname: {surname}
+const ProfileUI = ({ name, surname, profile_pic }) => (
+  <div className="div_profile-view">
```

Comment:
Guys, try to learn BEM better. It's a fundamental skill that will be required by most of the companies today.
I don't stress this point to stress you, just to help you prepare yourself.

Reply:
I've fixed it.